docs/guides/1.4.0-upgrade-guide.md

```diff
@@ -114,6 +121,175 @@ resource "couchbase-capella_network_peer" "new_network_peer" {
 ```
 For more information, see the [network peer examples](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/network_peer).
 
+## GSI
+To create a deferred index, use the `query_indexes` resource as follows:
+```
+locals {
+  index_template = templatefile("${path.module}/indexes.json", {
+    organization_id = var.organization_id
+    project_id      = var.project_id
+    cluster_id      = var.cluster_id
+  })
+
+  decoded_template = jsondecode(local.index_template)
+  index_names      = [for idx, details in local.decoded_template.resource["couchbase-capella_indexes"] : details.index_name]
+}
+```
+```
```

Comment:
Please mention this is example of `indexes.json`

Reply:
I filed https://jira.issues.couchbase.com/browse/AV-92582

This can be added to 1.4.1 docs.

Reply:
Sure @l0n3star